exercises/reduce_test.rb

```diff
@@ -15,51 +15,41 @@ def test_sum_a_list_of_numbers
   def test_subtract_list_of_numbers
     numbers = [28, 12, 38, 1, 91]
     result = numbers.reduce(0) do |difference, number|
-      # Your code goes here
+      difference -= number
     end
     assert_equal -170, result
   end
 
   def test_multiply_list_of_numbers
-    skip
     numbers = [2, 3, 5, 7]
-    # initial value is 1
-    # Your code goes here
+    result = numbers.reduce(1) { |acc, num| acc *= num }
```

Comment:
Useless assignment to variable - `acc`. Use just operator `*`.<br>Prefer `inject` over `reduce`.